src/native/runtime-base/logger.cc

```diff
@@ -239,8 +239,6 @@ Logger::init_logging_categories (char*& mono_log_mask, char*& mono_log_level) no
 #endif
 	}
 
-#if DEBUG
 	if ((log_categories & LOG_GC) != 0)
 		_gc_spew_enabled = 1;
```

Comment:
This would have to be changed to assign `true` now :)

Reply:
Likewise the `set_gc_spew_enabled` parameter needs to be changed to `bool`

Reply:
Why aren't we getting warnings about all this?